target/linux/ramips/image/mt76x8.mk

```diff
@@ -57,6 +61,19 @@ define Device/hc5661a
 endef
 TARGET_DEVICES += hc5661a
 
+define Device/ki_rb
+  DTS := ki_rb
+  IMAGE_SIZE := $(ralink_default_fw_size_16M)
```

Comment:
$(ralink_default_fw_size_16M) doesn't match the size of your firmware partition. ralink_default_fw_size_16M == 16121856 bytes == 0xf60000 bytes. Please use 16064k.

Reply:
Firmware partition size is 0xe90000 bytes. So now IMAGE_SIZE is 14912k. It's right?